### ISSUE TITLE
Do not sent plain text password in confirmation email

### ISSUE THREAD

**Issue by ramsey-darling1** [ubirimi/ubirimi#44]

**Comment by gileswells** [comment]
Agreed on this. Email is never a secure location for a password. If anything the activation email should include a one time url to set your first password or at least allow them to set it during user creation. These rules should also apply to password resetting if they do not already.
